src/styles/index.scss

```diff
@@ -2,2 +1,69 @@
+@import './variable';
+
+.page {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    height: $minutes-height;
+    width: $minutes-width;
+    background-color: $minutes-color;
+    bottom: 50%;
+    left: calc(50% - (#{$minutes-width} / 2));
+    transform-origin: bottom center;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__seconds {
+    position: absolute;
+    height: $seconds-height;
+    width: $seconds-width;
+    background-color: $seconds-color;
+    bottom: 50%;
+    left: calc(50% - (#{$seconds-width} / 2));
+    transform-origin: bottom center;
+    animation: rotate 60s linear infinite;
```

Comment:
There are discrepancies in the animation durations. The `animation` shorthand property for `.stopwatch__minutes` is set to `3600s`, but the `animation-duration` in `.stopwatch--speed .stopwatch__minutes` is `600s`. Similarly, for `.stopwatch__seconds`, the shorthand is `60s`, but the `animation-duration` is `10s`. Ensure these values are consistent to avoid unexpected behavior.